### PR TITLE
SMPC - First Offical Discussion

### PR DESCRIPTION
**Pull request by gmuraru**
## Description
Add the first discussion for the SMPC Team

### CODE REVIEW

File: cryptography/smpc/README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# Secure Multi-Party Computation Team
\ No newline at end of file
+# Secure Multi-Party Computation Team
+
+The SMPC team has a weekly meeting every Thrusday, at 15:00 CET Time.
```
Comment (LaRiffle) [inline cryptography/smpc/README.md:3]:
```suggestion
The SMPC team has a weekly meeting every Thursday, at 15:00 CET Time.
```

File: cryptography/smpc/meetings/10.10.2020.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+## Weekly Update
```
Comment (LaRiffle) [inline cryptography/smpc/meetings/10.10.2020.md:1]: 10.09.2020 ? :D

Comment (gmuraru) [reply cryptography/smpc/meetings/10.10.2020.md]: Living in the future 😎

**Review by iamtrask — APPROVED** [review]

**Review by LaRiffle — APPROVED** [review]
🙌

**Pull request merged**